components/global/IndicatorRx1dayCmip6.vue

```diff
@@ -10,43 +10,43 @@ const latLng = computed<LatLngValue>(() => placesStore.latLng)
 const layers: MapLayer[] = [
   {
     id: 'indicator_rx1day_historical_era',
-    title: '1980–2009, GFDL-ESM4',
+    title: '1980–2009, TaiESM1',
     source: 'rasdaman',
     wmsLayerName: 'cmip6_indicators',
     style: 'ardac_indicator_rx1day_historical_era',
     legend: 'rx1day',
-    rasdamanConfiguration: { dim_model: 4, dim_scenario: 0 },
+    rasdamanConfiguration: { dim_model: 11, dim_scenario: 0 },
     coastline: true,
   },
   {
     id: 'indicator_rx1day_midcentury_era',
-    title: '2040–2069, GFDL-ESM4, SSP5-8.5',
+    title: '2040–2069, TaiESM1, SSP5-8.5',
     source: 'rasdaman',
     wmsLayerName: 'cmip6_indicators',
     style: 'ardac_indicator_rx1day_midcentury_era',
     legend: 'rx1day',
-    rasdamanConfiguration: { dim_model: 4, dim_scenario: 4 },
+    rasdamanConfiguration: { dim_model: 11, dim_scenario: 4 },
     coastline: true,
   },
   {
     id: 'indicator_rx1day_latecentury_era',
-    title: '2070–2099, GFDL-ESM4, SSP5-8.5',
+    title: '2070–2099, TaiESM1, SSP5-8.5',
     source: 'rasdaman',
     wmsLayerName: 'cmip6_indicators',
-    style: 'ardac_indicator_rx1day_latecentury_era',
+    style: 'ardac_indicator_rx1day_latecentury',
```

Comment:
```suggestion
    style: 'ardac_indicator_rx1day_latecentury_era',
```

There isn't a style called ardac_indicator_rx1day_latecentury on Apollo